### ISSUE TITLE
find: "undefined" on npm install

### ISSUE THREAD

**Issue by Vebryn** [OpenZWave/node-openzwave-shared#151]
Hello,

I encount an issue very similar to #145. I'm also using Pi3. I install 1.3.2 package using this command :
`sudo /usr/bin/npm install --global openzwave-shared`

```
> openzwave-shared@1.3.2 preinstall /usr/local/lib/node_modules/openzwave-shared
> node lib/install-ozw.js
> openzwave-shared@1.3.2 install /usr/local/lib/node_modules/openzwave-shared
> node-gyp rebuild

find: "undefined": Aucun fichier ou dossier de ce type
gyp: Call to 'find undefined -name ZWSecurity.h | wc -l' returned exit status 0. while trying to load binding.gyp
```

I agree that OZW_INC variable seems wrong. I check pkg-config. Return code and list are fine.

```
$ pkg-config --exists libopenzwave
$ echo $?
0
$ pkg-config --list-all |grep libopenzwave
libopenzwave           libopenzwave - A Open Source implementation of the ZWave Serial API
```

I investigate ozwpaths.js. Directories are retrieved by pkg-config or find command. Both method are working. I don't really understand what is missing or wrong. Any idea ?

```
$ find /usr/local/include -name OZWException.h
/usr/local/include/openzwave/OZWException.h
$ pkg-config --variable=includedir libopenzwave
/usr/local/include/openzwave/
```

Best regards

**Comment by ekarak** [comment]
please try running: `pkg-config --variable=includedir libopenzwave` - I'm willing to bet it will return *an empty string*.  Once again the .pc file bundled with OpenZWave is broken. It should return the C++ headers inclusion path...

**Comment by ekarak** [comment]
I've released version 1.3.3, please update and retry (and sorry!)
